draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.spec.tsx

```diff
@@ -578,4 +578,93 @@ describe("<TitleBlockZen />", () => {
       expect(testOnClickFn).not.toHaveBeenCalled()
     })
   })
+
+  describe("automation ID behaviour", () => {
+    describe("when default automation IDs are not provided alongside required conditional renders", () => {
+      it("renders the default automation IDs", () => {
+        const automationdIds = {
+          titleAutomationId: "TitleBlock__Title",
+          avatarAutomationId: "TitleBlock__Avatar",
+          subtitleAutomationId: "TitleBlock__Subtitle",
+          sectionTitleAutomationId: "TitleBlock__SectionTitle",
+          sectionTitleDescriptionAutomationId:
+            "TitleBlock__SectionTitleDescription",
+          breadcrumbAutomationId: "TitleBlock__Breadcrumb",
+          breadcrumbTextAutomationId: "TitleBlock__BreadcrumbText",
+        }
```

Comment:
This relates to PR note that using an object in this test although keys are not used.